contrib/supervisord.conf

```diff
@@ -17,26 +17,32 @@ password = 🍟
 [rpcinterface:supervisor]
 supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
 
-[program:redis]
-command = redis-server
+[program:main]
+command = python3 manage.py runserver 8000
+environment = DJANGO_SETTINGS_MODULE="readthedocs.settings.main.dev"
 directory = ../
-priority = 0
-startretries = 0
+priority = 100
+stopasgroup = True
 stderr_logfile = /dev/stderr
 stderr_logfile_maxbytes = 0
 
-[program:main]
-command = python3 manage.py runserver 8000
+[program:web]
+command = python3 -m celery worker -A readthedocs.worker -Q web -l DEBUG -c 1
+environment = DJANGO_SETTINGS_MODULE="readthedocs.settings.main.dev"
 directory = ../
 priority = 100
 stopasgroup = True
 stderr_logfile = /dev/stderr
 stderr_logfile_maxbytes = 0
 
-[program:celery]
-command = python3 -m celery worker -A readthedocs.worker -Q default,celery,web,builder -l DEBUG -c 2
+[program:build]
+command = python3 -m celery worker -A readthedocs.worker -Q default,celery,builder -l DEBUG -c 1
```

Comment:
Did you change `-c` to `1` on purpose? Shouldn't it be `2` to be more similar to production?

Reply:
Is there any situation where 2 build workers is important locally?

This was previously all of the queues, so having 2 workers was important -- mostly for tasks on the web queue that are prone to race conditions. I don't think there is a strong need for more than one build queue worker though.

Reply:
I don't think it's important to have more than one build queue worker here, but if there is a strong argument for one, we can increase this. There should be multiple workers on the web process however, i'll update that to 2.